draft-steele-cose-hash-envelope.md

```diff
@@ -201,27 +192,31 @@ For example, when signing with ECDSA using P-256 and SHA-256, use SHA-256 to has
 
 # IANA Considerations
 
+## Requirements Notation
+
+{::boilerplate bcp14-tagged}
+
 ## COSE Header Algorithm Parameters
 
-* Name: payload hash algorithm
```

Comment:
* -> - seems non-consequential?

Reply:
The only change here was * to - for markdown formatting validation.

Was there another suggestion?